src/test/java/org/kframework/ctests/OuterTest.java

```diff
@@ -27,7 +27,8 @@
 
 public class OuterTest {
 
-    private String startPath = "src/test/k/kore_in_k/tests";
+    //private String startPath = "src/test/k/kore_in_k/tests";
+    private String startPath = "src/test/k/unparametric_examples/imp";
```

Comment:
Why did you change this path?
I was using this file to test the K grammar for the outer KORE. I wasn't doing anything more than that because analyzing and processing the input to create grammars involves a lot of work.
This is why to test IMP, I'm still using the old infrastructure (outer JavaCC parser and java+scala classes).